pom.xml

```diff
@@ -2099,12 +2100,6 @@
         <module>itests</module>
       </modules>
     </profile>
-    <profile>
```

Comment:
We have Iceberg profile in itests module https://github.com/apache/hive/blob/3f6f940af3f60cc28834268e5d7f5612e3b13c30/itests/pom.xml#L501 as well which runs tests in qtest-iceberg module. This also should be enabled by default

Reply:
Good catch! I checked the original ticket HIVE-25088.
Fixed in https://github.com/apache/hive/pull/5428/commits/8c1b26b13cd894085c2fab0a5d1b62c74432b757

Please help to check if this change is ok.
Thanks @Indhumathi27